README.md

```diff
@@ -124,3 +125,12 @@ Run all the tests:
 ```bash
 $ ./scripts/test
 ```
+
+## Processing Imagery
+
+In staging and production, a batch job will automatically be kicked off for processing after a
```

Comment:
I'd add a comment here that the API server needs to be running locally while performing this script, since the upload processing needs to pull the upload record and create corresponding records after processing is done through API server.

Reply:
Good catch! I didn't even realize this myself.

I ended up making the script itself require that `api-server` be running which I think is a bit nicer than having to read the docs.